OMEdit/OMEditLIB/OMS/OMSProxy.cpp

```diff
@@ -1013,12 +1013,13 @@ bool OMSProxy::initialize(QString cref)
  */
 bool OMSProxy::list(QString cref, QString *pContents)
 {
-  QString command = "oms_list";
+  QString command = "oms_exportSnapshot";
+  QString cref_ = cref + ":SystemStructure.ssd";
```

Comment:
For now, I hard-coded `SystemStructure.ssd`. Later, it should be possible to select any of the other files as well.